eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -254,8 +256,15 @@ function DeployStressPackage(
             Write-Host "Building and pushing stress test docker image '$imageTag'"
             $dockerFile = Get-ChildItem $dockerFilePath
 
-            Run docker build -t $imageTag -f $dockerFile $dockerBuildFolder
+            $dockerBuildCmd = @("docker"; "build"; "-t"; $imageTag;"-f";$dockerFile)
```

Comment:
Nit: It may be a little easier to read the command if using the simplified array syntax.

```suggestion
            $dockerBuildCmd = "docker", "build", "-t", $imageTag, "-f", $dockerFile
```